requirements.txt

```diff
@@ -0,0 +1,4 @@
+django-netjsonconfig>=0.8.0,<0.9.0
```

Comment:
We have moved this repository to pipenv[1].
Hence, `requirements.txt` is not useful.
Also, you may want to look in `.install-dev.sh` for now.

---
1. https://docs.pipenv.org/en/latest/